sources/packages/api/src-sql/EducationPrograms/Add-approval-columns.sql

```diff
@@ -0,0 +1,51 @@
+-- Add submitted_on to education_programs table.
+ALTER TABLE
+  sims.education_programs
+ADD
+  COLUMN IF NOT EXISTS submitted_on TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now();
```

Comment:
What is the significance of using the submitted_on? does it denote the date submitted for approval or program creation date.